extension.html

```diff
@@ -90,7 +121,7 @@ <h2>Extensions to the <code>PointerEvent</code> interface</h2>
 The dispatched event's attributes will be initalized in a way that is best representative of all the coalesced events.
 For example its <a href="https://www.w3.org/TR/pointerlock/#widl-MouseEvent-movementX">movementX</a> and <a href="https://www.w3.org/TR/pointerlock/#widl-MouseEvent-movementY">movementY</a> ([[!POINTERLOCK]]) COULD be the sum of those of all the coalesced events.
 None of the events in the sequence will have (nested) coalesced events, so <code>getCoalescedEvents</code> returns an empty sequence for them.
-This API always returns at least one coalesced event for <code>pointermove</code> events and an empty list for other types of <code>PointerEvents</code>.</p>
+This API always returns at least one coalesced event for <code>pointermove</code> and <code>pointerrawmove</code> events and an empty list for other types of <code>PointerEvents</code>.</p>
```

Comment:
Why we need this for pointerrawmove?

Reply:
I guess the same question would also hold for a pointermove that only has one coalesced event. Because in that case pointermove itself is that coalesced events. I believe the answer is that both pointerrawmove and pointermove may get coalesced. So in javascript code it is easier to always handle coalesced events of these events (if they care about that precision) instead of a check for the size of the coalesced events and if it is 0 then use the actual event itself. That is  the only reason I can think of. What do you think? Is this way more ergonomic and easy to use (like the sample drawing code later in this doc)?